### DISCUSSION TITLE
MDN pages load but disappearing on Firefox 91.8.0esr (64-bit)

### DISCUSSION THREAD

**Discussion by himanshugarg** [mdn/mdn-community#602] [Content] [answered]
This started only yesterday. I have since cleared everything in History, removed extensions, reinstalled the browser. Other websites open fine. MDN opens fine on Chrome

![Recording 2022-04-22 at 05 45 20](https://user-images.githubusercontent.com/2392803/164572259-982dec2e-7b7f-446f-aaf7-f110183b084a.gif)

**Comment by hamishwillee** [comment, marked as answer]
Thanks. Could this be a duplicate of https://github.com/mdn/yari/issues/6035 ?

**Comment by pepelsbey** [comment]
Closing as answered/settled.
